### PR TITLE
Upmerge v0.34 to edge

### PR DESCRIPTION
**Pull request by willtsai**
Thank you for helping make the Radius documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.radapp.io/community/contributing/docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

upmerging changes from `v0.34` to `edge`

### CODE REVIEW

File: docs/config.toml
Comment (Reshrahim) [inline docs/config.toml:1]: I think these changes shd be reverted back as they update the config of edge website.

File: docs/config.toml [hunk 1]
```diff
@@ -69,15 +69,15 @@ tag = "tags"
...
+chart_version = "0.34.0"
 
 # Algolia Search
 algolia_docsearch = true
 
 # GitHub Information
 github_repo = "https://github.com/radius-project/docs"
 github_subdir = "docs"
-github_branch = "edge"
+github_branch = "v0.34"
```
Comment (willtsai) [inline docs/config.toml]:
```suggestion
github_branch = "edge"
```

File: docs/config.toml [hunk 2]
```diff
@@ -69,7 +69,7 @@ tag = "tags"
 copyright = "The Radius Authors. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see the https://linuxfoundation.org/trademark-usage/ page."
 version = "edge"
 tag_version = "latest"
-chart_version = "0.26.0"
+chart_version = "0.34.0"
```
Comment (willtsai) [inline docs/config.toml]:
```suggestion
chart_version = "0.26.0"
```

File: docs/config.toml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-baseURL = "https://edge.docs.radapp.io/"
+baseURL = "https://docs.radapp.io/"
```
Comment (willtsai) [inline docs/config.toml]:
```suggestion
baseURL = "https://edge.radapp.io/"
```

File: docs/config.toml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-baseURL = "https://edge.docs.radapp.io/"
+baseURL = "https://edge.radapp.io/"
```
Comment (willtsai) [inline docs/config.toml]:
```suggestion
baseURL = "https://edge.docs.radapp.io/"
```

File: docs/layouts/partials/hooks/body-end.html [hunk 5]
```diff
@@ -5,7 +5,7 @@
     container: '#docsearch',
     appId: 'ED67NSVSZS',
     apiKey: '21e2920c3a9c29b98cbc382694413f29',
-    indexName: 'radapp-dev-edge',
+    indexName: 'radapp-dev',
```
Comment (Reshrahim) [inline docs/layouts/partials/hooks/body-end.html]:
```suggestion
    indexName: 'radapp-dev-edge',
```

File: .github/workflows/issues.yml [hunk 6]
```diff
@@ -46,4 +46,4 @@ jobs:
           ado_new_state: 'New'
           ado_active_state: 'Active'
           ado_close_state: 'Closed'
-          ado_wit: 'GitHub Issue'
+          ado_wit: 'GitHub Issue'
```
Comment (willtsai) [inline .github/workflows/issues.yml:49]:
```suggestion
          ado_wit: 'GitHub Issue'
```

File: docs/config.toml
Comment (willtsai) [reply docs/config.toml:1]: yup...reverted

**Review by Reshrahim — APPROVED** [review]
LGTM

**Pull request closed without merging**